src/loggers.jl

```diff
@@ -133,9 +133,13 @@ getlevels(logger::Logger) = logger.levels
 """
     register(::Logger)
 
-Register an existing logger with Memento.
+Register an existing logger with Memento if it has not already been registered.
 """
-register(logger::Logger) = _loggers[logger.name] = logger
+function register(logger::Logger)
+    if !in(logger.name, keys(_loggers))
+        _loggers[logger.name] = logger
+    end
```

Comment:
Maybe we should log a debug message if a logger is already registered? Memento should probably register a logger of its own to help with debugging in the future.